docs/plugin/Leaflet.Control.Opacity/dist/L.Control.Opacity.css

```diff
@@ -1 +1 @@
-input[type='range']{-webkit-appearance:none;-webkit-tap-highlight-color:rgba(255,255,255,0);width:110px;height:10px;margin:0;border:none;padding:1px 2px;border-radius:30px;background:#f1f0ee;outline:none}input[type='range']::-ms-track{border:inherit;color:transparent;background:transparent}input[type='range']::-ms-fill-lower,input[type='range']::-ms-fill-upper{background:transparent}input[type='range']::-ms-tooltip{display:none}input[type='range']::-ms-thumb{width:15px;height:18px;border-radius:12px;border:0;background-image:linear-gradient(to bottom,#1253a4 0,#1253a4 100%)}.leaflet-control-layers-label{margin:0 0 8px 1px}
```

Comment:
Without my patch any kind of `input[type-'range']` HTML tag is styled by your css. I just add a scope of this style. Please, I hope is enough to merge my PR.